### ISSUE TITLE
Does it work with contents dynamically loaded?

### ISSUE THREAD

**Issue by dev-win911** [GuOrg/Gu.Wpf.UiAutomation#171]
We have an application where content is dynamically loaded in the Content controller and things are put together using MEF. So it seems like I can never get hold of elements that are loaded dynamically.
So, my question is does this tool work in those scenarios?
I think it can work if I can reload or refresh certain parts of control. Is there an option to do it?

**Comment by JohanLarsson** [comment]
No idea but it might, you'd have to try it.

**Comment by dev-win911** [comment]
Yeah i have tried it and seems like it does not work. MainWindow object doesn't seems to have any information about this dynamically loaded objects. So I was wondering if there is a method to reload the certain control? In this senario if i can refresh/reload the children of main window it might work.
